static/templates/pod.html

```diff
@@ -0,0 +1,12 @@
+<div class="panel panel-default">
```

Comment:
Thanks for noting :) Now that we are adding action support, I think it is likely that we will change how this looks. In praekelt/casepro#48 (one of the smaller PRs for the next PR we plan on making later this sprint), in order to fit action buttons into the UI, we're changing it to look something like this (might still change a bit):

![pod-with-actions](https://cloud.githubusercontent.com/assets/1731223/17019564/3c757e60-4f3d-11e6-943a-a1fa8991ab56.png)

Thought it would be worth mentioning now to avoid inconsistencies with the contact panel.